Makefile.include

```diff
@@ -833,6 +833,8 @@ objdump:
 	$(call check_cmd,$(OBJDUMP),Objdump program)
 	$(OBJDUMP) $(OBJDUMPFLAGS) $(ELFFILE) | less
 
+# inlcude clang-tidy
+include $(RIOTMAKE)/clang_tidy.inc.mk
```

Comment:
Why not only include this if llvm toolchain is used (there's also a typo btw):

```suggestion
# include clang-tidy if llvm toolchain is used
ifeq (llvm,$(TOOLCHAIN))
  include $(RIOTMAKE)/clang_tidy.inc.mk
endif
```

Then there's no need for the error message in `clang_tidy.inc.mk` (that I would rename `clang-tidy.inc.mk` but that's a matter of taste and I won't insist)

Reply:
I thought it made the target more visible that way.